dashboard/alerts/alert.py

```diff
@@ -86,30 +127,36 @@ def saveAlert(self):
         :return:
         """
         try:
-            # Open config file to get path
-            config_file = open('/etc/capstone-ids/config.json', 'r')
-            config_data = json.load(config_file)
-            path = config_data["alert_collection_path"]
-
-            # Default path
-            if path == "" or path is None:
-                path = "/etc/capstone-ids/alert_collection.json"
+            path = CONFIG.alert_collection_path
```

Comment:
I've update the config with the new value, but am a little confused on the proper way to access the config object. Where is the config object being initialized? I can't extract the values without initializing the class if I just import the class and am running into so issues with pulling the values because of that. I've added a static CONFIG value to the config class to compensate, but let me know if I'm doing this incorrectly.

Reply:
Addressed in discord but currently the object lives in cids_main.py, this may be moved in a future PR once the directory structure is fixed to an init.py file but for now use the following line
`from cids_main import run_config`